api/tests/routes/pro/test_offerers_addresses.py

```diff
@@ -0,0 +1,300 @@
+                },
+            )
+
+        assert response.status_code == 201
+        content = response.json
+        address = geography_models.Address.query.one()
+        assert content["label"] == expected_data["label"]
+        assert content["offererId"] == offerer_id
+        assert content["address"]["id"] == address.id
+        assert content["address"]["inseeCode"] == expected_data["inseeCode"] == address.inseeCode
+        assert content["address"]["street"] == expected_data["street"] == address.street
+        assert content["address"]["postalCode"] == expected_data["postalCode"] == address.postalCode
+        assert content["address"]["city"] == expected_data["city"] == address.city
+        assert content["address"]["latitude"] == expected_data["latitude"] == float(address.latitude)
+        assert content["address"]["longitude"] == expected_data["longitude"] == float(address.longitude)
+
+    @pytest.mark.usefixtures("db_session")
+    def test_user_cant_create_offerer_address_on_foreign_offerer(self, client):
+        pro_user = users_factories.ProFactory()
+        offerer = offerers_factories.OffererFactory()
+        offerers_factories.UserOffererFactory(user=pro_user, offerer=offerer)
+        foreign_offerer = offerers_factories.OffererFactory()
+        foreign_offerer_id = foreign_offerer.id
+
+        expected_data = {
+            "label": "Label",
+            "banId": "75108_5194_00089",
+            "inseeCode": "75108",
+            "street": "89 Rue de la Boétie",
+            "postalCode": "75008",
+            "city": "Paris",
+            "latitude": 48.87171,
+            "longitude": 2.30829,
+        }
+
+        http_client = client.with_session_auth(pro_user.email)
+
+        # Fetch the session
+        # Fetch the user
+        # Check permission
+        with assert_num_queries(3):
+            response = http_client.post(
+                f"/offerers/{foreign_offerer_id}/addresses/",
+                json={
+                    "label": expected_data["label"],
+                    "street": expected_data["street"],
+                    "inseeCode": expected_data["inseeCode"],
+                },
+            )
+
+        assert response.status_code == 403
+        assert not geography_models.Address.query.count()
+        assert not offerers_models.OffererAddress.query.count()
+
+    @override_settings(ADRESSE_BACKEND="pcapi.connectors.api_adresse.ApiAdresseBackend")
+    @pytest.mark.usefixtures("db_session")
+    def test_check_consistency_betwenn_address_and_coordinates(self, client, requests_mock):
+        payload = {
+            "label": "Label",
+            "banId": "75118_2974_00018",
+            "inseeCode": "75118",
+            "street": "18 Rue Duhesme",
+            "postalCode": "75018",
+            "city": "Paris",
+            "latitude": 48.89078,
+            "longitude": -15.30829,  # Dans l'océan atlantique
+        }
```

Comment:
Je pense qu'on peut simplifier le payload:
```suggestion
        payload = {
            "label": "Label",
            "inseeCode": "75118",
            "street": "18 Rue Duhesme",
        }
```
Le payload tel qu'il est peut porter à confusion, quelqu'un qui va lire rapidement ce test, pensera qu'on attend toute ces informations comme entrées

Reply:
yes bien vu merci